### PR TITLE
New design

### PR DESCRIPTION
**Pull request by domiin50**
PR created just for a review of the styling of the whole first version of the new design.

### CODE REVIEW

File: server/apiCallbacks.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ export const getLoggedUserInfo = (req, res) => {
 export const getIssueList = async (req, res) => {
   const getHandler = (resolve, reject) => (error, data) => {
     if (error) {
-      reject(JSON.stringify(error))
+      // reject(JSON.stringify(error))
```
Comment (mikx98) [inline server/apiCallbacks.js]: remove

File: src/tracker/JiraSidebar/IssueInput.js [hunk 2]
```diff
@@ -3,20 +3,20 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { compose, withHandlers } from 'recompose'
 import { withStyles } from '@material-ui/core/styles'
-import debounce from 'es6-promise-debounce'
 
+import AsyncSelect from 'react-select/lib/Async'
+//import TextField from '@material-ui/core/TextField'
```
Comment (mikx98) [inline src/tracker/JiraSidebar/IssueInput.js]: remove

File: server/apiCallbacks.js (hunk 1 above)
Comment (domiin50) [reply server/apiCallbacks.js]: Resolved in future branch

File: src/tracker/JiraSidebar/IssueInput.js (hunk 2 above)
Comment (domiin50) [reply src/tracker/JiraSidebar/IssueInput.js]: Resolved in future branch

**Pull request closed without merging**